config/manager/manager.yaml

```diff
@@ -24,7 +24,7 @@ spec:
 apiVersion: apps/v1
 kind: StatefulSet
 metadata:
-  name: controller-manager
```

Comment:
I think this controller controls not only machine but also cluster, how about rename it to `clusterapi-controllers` same as openstack https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/config/manager/deployment.yaml#L4